src/app/lib/utilities/describeDuration/index.js

```diff
@@ -0,0 +1,29 @@
+const defaultTranslations = {
+  s: '{x} second',
+  ss: '{x} seconds',
+  m: '{x} minute',
+  mm: '{x} minutes',
+  h: '{x} hour',
+  hh: '{x} hours',
+};
+
+const humanize = (labels, key, num) => {
+  if (num === 1) {
+    return `${labels[key].replace('{x}', num)} `;
+  }
+  if (num > 1) {
+    return `${labels[key.repeat(2)].replace('{x}', num)} `;
+  }
+  return '';
+};
+
+const describeDuration = (duration, translations = defaultTranslations) => {
+  const hrs = humanize(translations, 'h', duration.hours());
```

Comment:
this would need to be `asHours` right? it would return the wrong values for durations over 24 hrs

`moment.duration({ hours: 48 }).hours()` returns `0`

Reply:
Yes. I will also need to add unit tests to catch this. But will do that once we settle on which approach to take. Thanks for pointing that out.